src/types/workshopContext.ts

```diff
@@ -0,0 +1,56 @@
+import { IConfigDefinition, IConfigDefinitionField } from "./configDefinition";
+import { IAsyncStatus } from "./loadingState";
+import { IVariableType, IStructVariableFieldTypes } from "./variableTypes";
+
+export interface ExecutableEvent {
+    executeEvent: () => void;
```

Comment:
even if we don't immediately implement different states let's use the target API
```suggestion
    executeEvent: IAsyncLoaded<() => void>;
```

Reply:
The API for executing an event [here](https://github.palantir.build/foundry/forge/blob/develop/packages/workshop/object-analytics-config-spec/src/api/types.ts#L100) is 
```
execute: (mouseEvent: IMouseEvent | React.KeyboardEvent | undefined) => boolean;
```

So in the plugin should it be this instead? 
```
executeEvent: (mouseEvent: IMouseEvent | React.KeyboardEvent | undefined) => boolean;
```